open-craft/xblock-pdf
# feat: add French translations [BB-4504]

This ports fr_CA translations from [raccoongang/xblock-pdf](https://github.com/raccoongang/xblock-pdf).

## Testing instructions
1. Install this XBlock and add "pdf" to course advanced modules in Studio.
2. Go to [/update_lang/](http://localhost:18000/update_lang/) and change your language to `fr`.
3. View the unit in the LMS and check that the translations are working correctly.
